project-icp/bee-pollinator-app
# Update CDL crop group names

## Overview

Updated per client document.

Regenerate derivative files.

Connects to #218 

### Demo

![image](https://user-images.githubusercontent.com/1042475/28123536-2bf795f8-66ef-11e7-98c2-96869e2e681f.png)

## Testing Instructions

- Verify that the values in the legend match those specified in https://docs.google.com/spreadsheets/d/1uA4vXIvQvS3HFqXLamVYPhhPVp4P-xyPvzbG8QUBe3Q/edit?ts=591e024b#gid=0.
